### DISCUSSION TITLE
Tooltip for a decorative icon

### DISCUSSION THREAD

**Discussion by vankhoawin** [twilio-labs/paste#853] [Q&A] [answered]
Hey team, I've noticed in the Tooltip docs that reads: `Do not place tooltips on non-focusable elements, like an icon. Wrap them in a focusable element and place the tooltip on that.`

In a page we own that predates Paste, we do just that:

![image](https://user-images.githubusercontent.com/6936736/96039903-425b0080-0e1e-11eb-9d97-4da53c4f3626.png)

What should the wrapped element be, if not a plain icon? Should this be a link or button, or something else?

**Comment by ghostling** [comment, marked as answer]
https://github.com/twilio-labs/paste/discussions/683#discussioncomment-56645

**Reply by SiTaggart** [reply]
Yup, as @ghostling points out, this has come up before and been answered in a previous discussion

**Reply by vankhoawin** [reply]
I've also heard that the reset button might be removed? Can you confirm this or should I go ahead and wrap using a reset button

**Reply by SiTaggart** [reply]
Keep going for now. If we ship a replacement for it we'll make the migration as painless as possible. It might be the same API but a different implementation under the hood

**Comment by aayushpi** [comment]
@vnguyen94 In this case, the icons don't have any action at all. They're used to convey information visually.

My recommendation here is to:

1. [Set the icon as not decorative](https://paste.twilio.design/icons/usage-guidelines/#specify-if-an-icon-is-decorative)
2. [Pair this icon with text](https://paste.twilio.design/icons/usage-guidelines/#give-informative-icons-a-title) so that a user does not have to guess the meaning of an icon
